REQUIRE

```diff
@@ -3,3 +3,5 @@ LightGraphs 0.9.0
 JuMP 0.13.2
 MatrixDepot
 BlossomV 0.1
+Hungarian 0.1.2
```

Comment:
we shouldn't depend on patches, only versions x.y, not x.y.z

Reply:
The thing is that a specific functionality was added in this version (handling `missing` values).

Reply:
ok, if we use version x.y this should fetch x.y.z with the biggest z
Otherwise as mentioned Hungarian 0.2 will do the job, also we can update JuMP to v0.18 I think